tests/benchdnn/common.cpp

```diff
@@ -243,12 +244,61 @@ static void zfree_protect(void *ptr) {
     ::free(ptr_start);
 }
 #endif
+struct memory_registry_t {
+    void add(void *ptr, size_t size) {
+        std::lock_guard<std::mutex> g(m_);
+        allocations_.emplace(std::pair<void *, size_t>(ptr, size));
+        total_size_ += size;
+
+        BENCHDNN_PRINT(6,
```

Comment:
Let's put v8, v6 would explode graph's typical usage since v7 prints a dumped graph which is handy usually.